plugins/package-managers/spm/src/main/kotlin/Spm.kt

```diff
@@ -84,20 +84,20 @@ class Spm(
 
         return when (definitionFile.name) {
             PACKAGE_SWIFT_NAME -> resolveLibraryDependencies(definitionFile)
-            else -> resolveAppDependencies(definitionFile)
+            else -> resolveLockfileDependencies(definitionFile)
         }
     }
 
     /**
      * Resolves dependencies when only a lockfile aka. `Package.Resolved` is available. This commonly applies to e.g.
      * Xcode projects which only have a lockfile, but no `Package.swift` file.
      */
-    private fun resolveAppDependencies(definitionFile: File): List<ProjectAnalyzerResult> {
-        val resolved = definitionFile.inputStream().use { json.decodeFromStream<PackageResolved>(it) }
+    private fun resolveLockfileDependencies(packageResolvedFile: File): List<ProjectAnalyzerResult> {
```

Comment:
Commit message: The function signature as such is not change, it's still the same amount of parameters of the same type. It's just that the function and parameter are renamed.